sparklingpandas/pcontext.py

```diff
@@ -175,47 +220,80 @@ def frame_to_rows(frame):
         return sp_df
 
     def sql(self, query):
-        """Perform a SQL query and create a L{Dataframe} of the result."""
+        """Perform a SQL query and create a L{Dataframe} of the result.
+        The SQL query is run using the local Spark SQL context. This
```

Comment:
What does local mean here? I am concerned people will think that the spark master is local. Maybe explain this without using this word?